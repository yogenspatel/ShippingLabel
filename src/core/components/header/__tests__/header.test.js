/* eslint-disable no-undef */

import React from "react";
import ReactDOM from "react-dom";
import Header from "../index";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("Header Component renders correctly", () => {
  it("Header renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Header />, div);
  });
  it("Header Component with snapshot", () => {
    const tree = shallow(<Header />);
    expect(toJson(tree)).toMatchSnapshot();
  });

});
