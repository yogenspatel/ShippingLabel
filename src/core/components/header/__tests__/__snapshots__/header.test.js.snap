// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders correctly Header Component with snapshot 1`] = `
<header
  className="navbar navbar-expand-lg navbar-dark bg-primary"
>
  <h1
    className="h1 text-white m-auto"
  >
    Shipping Label Maker
  </h1>
</header>
`;
